projects/demo/src/modules/experimental/pin/examples/1/index.html

```diff
@@ -0,0 +1,26 @@
+<!-- eslint-disable @html-eslint/no-inline-styles -->
+<div tuiPinGroup>
+    <tui-pin
+        src="assets/images/avatar.svg"
```

Comment:
Why is avatar an SVG? It should support raster images too.

Reply:
But tui-svg doesn't support raster image

Reply:
That means we cannot just use `tui-svg`. Perhaps it's better to externalize it so people can write both:
```html
<tui-pin><tui-svg src='...'></tui-svg></tui-pin>
<tui-pin><img src='...' /></tui-pin>
```

Reply:
fixed